tests/procs/test_executables.py

```diff
@@ -13,7 +13,10 @@
 
 def test_get_possible_names():
     env = Env(PATHEXT=[".EXE", ".COM"])
-    assert get_possible_names("file", env).sort() == ["file", "file.exe", "file.com", "FILE", "FILE.EXE", "FILE.COM"].sort()
+    assert (
+        get_possible_names("file", env).sort()
+        == ["file", "file.exe", "file.com", "FILE", "FILE.EXE", "FILE.COM"].sort()
+    )
```

Comment:
this can possibly be simplified by storing only lowercased executable names in the dictionary and match the "str".lower() against to compare it.